feed/views.py

```diff
@@ -1,3 +1,88 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http.response   import JsonResponse
+from django.views           import View
+
+from .models                import Feed, Comment, ImageUrl
+from user.models            import User
+from product.models         import Product
+
+class FeedView(View):    
+    def get(self, request):
+        MAXIMUM_COMMENT = 2
+        feed_list   = []
+        offset      = int(request.GET.get('offset'))
+        limit       = int(request.GET.get('limit'))
+    
+        for feed in Feed.objects.all().order_by('-id'):
+
+            # 피드 정보
+            feed_basic_data = {
+                'feed_id'         : feed.id, 
+                'feed_user'       : feed.user.nickname,
+                'created_at'      : feed.created_at,
+                'description'     : feed.description,
+                'like_number'     : feed.like_number,
+                'tag_item_number' : feed.tag_item_number,
+                'feed_main_image' : feed.imageurl_set.values('image_url').first()
+                }
+
+            # 피드의 상품 정보
+            if feed.product_id:
+                product_feed_id = feed.product
+                product_id      = feed.product.id 
+                product_name    = feed.product.name
+                discount_rate   = feed.product.discount_rate
+                price           = feed.product.price
+                product_image   = feed.product.productimageurl_set.get(is_main=1).image_url
+
+                product_data = {
+                    'product_name'   : product_name,
+                    'discount_rate'  : discount_rate,
+                    'price'          : price,
+                    'product_image'  : product_image
+                    }
+            else:
+                product_data = {
+                    'product_name'   : '',
+                    'discount_rate'  : '',
+                    'price'          : '',
+                    'product_image'  : ''
+                }
+            
+            # 피드의 댓글
+            if feed.comment_set.exists():
+                feed_comment_count      = feed.comment_set.all().count()                   
```

Comment:
```suggestion
                feed_comment_count      = feed.comment_set.count()
```
all() 메소드 없이도 count() 바로 사용하실 수 있습니다 :)

Reply:
여기도 마찬가지로 `feed_comment_count`라는 변수가 67번 줄에서 단 한 번밖에 쓰이지 않기 때문에 굳이 여기서 선언하지 않고 67번줄에서 바로 count()메소드 사용하셔도 될 것 같습니다 :)

Reply:
그 부분까지는 생각 못했는데 하나 또 배웁니다. 감사합니다 소헌님 ~ !!!!